webv2/updateTableSchema/addColumn.go

```diff
@@ -0,0 +1,34 @@
+package updateTableSchema
```

Comment:
Add the license header for all the new files

Reply:
License added to new files in package table